packages/react-component-library/src/components/DescriptionList/DescriptionList.stories.tsx

```diff
@@ -0,0 +1,64 @@
+import React from 'react'
+import { ComponentStory, ComponentMeta } from '@storybook/react'
+
+import { DescriptionList, DescriptionListItem } from './index'
+
+export default {
+  component: DescriptionList,
+  subcomponents: { DescriptionListItem },
+  title: 'Data List',
```

Comment:
Think this needs updating:

```suggestion
  title: 'Desscription List',
```